spdx/writers/tagvalue.py

```diff
@@ -368,7 +374,8 @@ def write_document(document, out, validate=True):
         write_snippet(snippet, out)
         write_separators(out)
 
-    out.write("# Extracted Licenses\n\n")
-    for lic in sorted(document.extracted_licenses):
-        write_extracted_licenses(lic, out)
-        write_separators(out)
+    if document.extracted_licenses:
+        out.write("# Extracted Licenses\n\n")
+        for lic in sorted(document.extracted_licenses):
+            write_extracted_licenses(lic, out)
+            write_separators(out)
```

Comment:
Why are extracted licenses handled differently than annotations and reviews? (two line breaks vs. one line break between each entry)

Reply:
Right, I overlooked that. They should be treated as the others.